### PR TITLE
New sample: Display route layer

### PR DESCRIPTION
**Pull request by williambohrmann3**
# Description

Create a new sample display route layer.

## Type of change

- New sample implementation

## Platforms tested on

- [x] WPF .NET 6
- [x] WPF Framework
- [x] WinUI
- [x] MAUI WinUI
- [x] MAUI Android
- [x] MAUI iOS
- [x] MAUI MacCatalyst

## Checklist

- [x] Self-review of changes
- [x] All changes work as expected on all affected platforms
- [x] There are no warnings related to changes
- [x] Code is commented and follows .NET conventions and standards
- [x] Codemaid and XAML styler extensions have been run on every changed file
- [x] No unrelated changes have been made to any other code or project files
- [x] Screenshots are correct size and display in description tab (800 x 600 on Windows, 600 height mobile screenshots for MAUI)

### CODE REVIEW

**Comment by williambohrmann3** [comment]
Did not want to take MAUI screenshot in Android considering that we're switching to Windows for screenshots - awaiting screenshot tool.

**Review by duffh — CHANGES_REQUESTED** [review]

File: src/MAUI/Maui.Samples/Samples/Layers/DisplayRouteLayer/DisplayRouteLayer.xaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+                IsVisible="False"
+                MaximumHeightRequest="75"
+                Style="{StaticResource EsriSampleControlPanel}"
+                WidthRequest="350">
+            <VerticalStackLayout>
+                <Label FontAttributes="Bold"
+                       Text="Display directions:"
+                       TextColor="{AppThemeBinding Light={StaticResource Black},
+                                                   Dark={StaticResource White}}" />
+                <Switch HorizontalOptions="Start" Toggled="Switch_Toggled" />
```
[the remark this answers is not in the document]
Comment (williambohrmann3) [reply src/MAUI/Maui.Samples/Samples/Layers/DisplayRouteLayer/DisplayRouteLayer.xaml]: In MAUI I think it makes sense to always display the directions. The show/hide toggle doesn't make sense on a mobile platform where the `SampleControlPanel` style causes the `MapView` to be resized.

**Review by amweis226 — COMMENTED** [review]
The layer not being shown in it's entirety is somewhat jarring, but also relatively difficult to fix given the symbology used (particularly the green "start" symbol). Would probably help to zoom out slightly when you take your final screenshots. Other than this and some minor suggested grammatical changes, it looks good. Good job.

File: src/MAUI/Maui.Samples/Samples/Layers/DisplayRouteLayer/DisplayRouteLayer.xaml (hunk 1 above)
[the remark this answers is not in the document]
Comment (duffh) [reply src/MAUI/Maui.Samples/Samples/Layers/DisplayRouteLayer/DisplayRouteLayer.xaml]: We use the show/hide logic on other mobile samples to my knowledge. The `SampleControlPanel` does not need to cause the `MapView` to be resized.

File: src/MAUI/Maui.Samples/Samples/Layers/DisplayRouteLayer/readme.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+## Use case
+
+Routes can be stored as feature collection layers. These layers can store useful information such as directions, estimated trip time, and more.
+
+You can create a route layer in ArcGIS Pro and store a route layer as a portal item, making it easy to access, share, or display.
+
+## How to use the sample
+
+Pan and zoom to view the route displayed by the feature collection layer. Toggle to view or hide the turn by turn directions.
+
```
[the remark this answers is not in the document]
Comment (williambohrmann3) [reply src/MAUI/Maui.Samples/Samples/Layers/DisplayRouteLayer/readme.md]: nice catch!

**Review by duffh — APPROVED** [review]

**Pull request merged**